var/spack/repos/builtin/packages/py-gluoncv/package.py

```diff
@@ -14,21 +14,26 @@ class PyGluoncv(PythonPackage):
     vision."""
 
     homepage = "https://gluon-cv.mxnet.io/"
-    url = "https://github.com/dmlc/gluon-cv/archive/v0.6.0.tar.gz"
+    pypi = "gluoncv/gluoncv-0.6.0.tar.gz"
+    git = "https://github.com/dmlc/gluon-cv.git"
 
-    version("0.6.0", sha256="5ac89d73f34d02b2e60595a5cc35f46d0a69376567fae3a9518005dd89161305")
+    version(
+        "0.10.5.post0", sha256="4598b9612e8b459a5a14ebeffedefcdae4a5700302a91f9b99fc82e9b08928a5"
+    )
+    version("0.6.0", sha256="313848b939c30e9e4c0040078421c02e32a350b8ebf2a966313fd893d7b3bdf6")
 
     depends_on("py-setuptools", type="build")
+
     depends_on("py-numpy", type=("build", "run"))
     depends_on("py-tqdm", type=("build", "run"))
     depends_on("py-requests", type=("build", "run"))
     depends_on("py-matplotlib", type=("build", "run"))
     depends_on("py-portalocker", type=("build", "run"))
     depends_on("pil", type=("build", "run"))
     depends_on("py-scipy", type=("build", "run"))
-    depends_on("py-cython", type="build")
-
-    patch("no-unicode-readme.patch")
+    depends_on("py-yacs", when="@0.8:", type=("build", "run"))
+    depends_on("py-pandas", when="@0.9:", type=("build", "run"))
+    depends_on("py-pyyaml", when="@0.9:", type=("build", "run"))
+    depends_on("py-autocfg", when="@0.9:", type=("build", "run"))
 
-    def install_options(self, spec, prefix):
-        return ["--with-cython"]
+    patch("no-unicode-readme.patch", when="@0.6.0")
```

Comment:
I actually have no idea why this patch was needed in the first place. It is not relevant for installing the package but just changes one line containing unicode in the `README.md`. The unicode is still there in 0.10.5.post0.

Reply:
It was needed to build 0.6.0 with Python 2 because `setup.py` reads the `README.md` to generate the `long_description`. But since we no longer support Python 2, I don't think the patch is needed for any version.

Reply:
I have removed the patch now and both versions install fine with `--test=root` on Fedora37 and gcc@12.3.1